### ISSUE TITLE
python3 support?

### ISSUE THREAD

**Issue by Jzarecta** [mrsmn/coinmarketcap-api#11]
I am getting the following bug:
```
  File "setup.py", line 16, in <module>
    long_description = file('README.md','r').read(),
NameError: name 'file' is not defined
```

**Comment by Jzarecta** [comment]
Fixed it by replacing it with open, but urllib2 also doesnt help. There should be something like six, to fix those modules incompatibilities.

**Comment by mrsmn** [comment]
Hi,

I probably broke python3 support  when I modified the library to use the official coinmarketcap API. I'll look into it in the coming days if I'm not too busy.

Cheers.

**Comment by Jzarecta** [comment]
Well urllib2 is python2 and thats what you got on core,py. You should try to use six package or a simple conditional to guess the version and work with the other modules:
https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection
https://docs.python.org/3/howto/urllib2.html

**Comment by mrsmn** [comment]
Something like this should do the trick (I've not tested this yet):

```python
try:
    import urllib.request as urllib2
except ImportError:
    import urllib2
```

**Comment by Jzarecta** [comment]
umm, is not a good practice to create unecessary errors/exceptions, you can do something similar with what I sent you on the link:
```
import sys

if sys.version[0] == 3:
    from importlib import abc
else:
    from importlib2 import abc
```

**Comment by mrsmn** [comment]
Fixed in 34faa2371cbf7d66688a4d876c046be737592864
